modules/checking-cluster-resource-availability-and-utilization.adoc

```diff
@@ -0,0 +1,41 @@
+// Module included in the following assemblies:
+//
+// *installing/validating-an-installation.adoc
+
+[id="checking-cluster-resource-availability-and-utilization_{context}"]
+= Checking cluster resource availability and utilization
+
+{product-title} provides a comprehensive set of monitoring dashboards that help you understand the state of cluster components.
+
+In the *Administrator* perspective you can access dashboards for core {product-title} components, including:
```

Comment:
s/perspective you can access/perspective, you can access/

Reply:
Resolved.